### DISCUSSION TITLE
Can't register a number, getting unrecognized arguments: '-a'

### DISCUSSION THREAD

**Discussion by kovinet** [AsamK/signal-cli#818] [Q&A] [answered]
Hello

I installed signal-cli and when trying to register a new account i am getting an error, that `-a` is unrecognized argument.

I am following the instruction in Quick start:

```
  signal-cli -a ACCOUNT register
```

I am entering my phone number with +9... format as ACCOUNT. I am missing something?

**Comment by kovinet** [comment]
It seems like correct argument is -u now, not -a (as stated in README.md)

**Comment by piramiday** [comment, marked as answer]
indeed, the correct flag has been `-u` until a couple of days ago.
now, starting with v0.10.0, the proper flag is `-a`, with the older `-u` kept for a while for backward compatibility.

> Renamed -u and --username flags to -a and --account to prevent confusion with upcoming Signal usernames. The old flags are also still supported for now.

@ https://github.com/AsamK/signal-cli/blob/master/CHANGELOG.md#0100---2021-12-11

so, unless you were compiling from source, at the time of your question you were running `signal-cli` which was at most version 0.9.2.
in general, pay attention to the last changes on the repo that might have not been released yet! :wink:

**Reply by kovinet** [reply]
You are right, I have 0.9.2. I have installed it according to instructions and I assumed I have the latest version. Good to know anyway, so I will be careful when I update it :)
